packages/lit-auth-client/src/lib/lit-auth-client.ts

```diff
@@ -138,8 +143,7 @@ export class LitAuthClient {
     }
 
     // Set RPC URL
-    this.rpcUrl =
-      options?.rpcUrl || 'https://lit-protocol.calderachain.xyz/replica-http';
+    this.rpcUrl = options?.rpcUrl || LIT_RPC.VESUVIUS;
```

Comment:
As with other, this will change the default for existing users of the SDK using manzano/habanero -- I think we have to make this decide the default URL based on `network` to be backwards compatible/non-breaking